### DISCUSSION TITLE
Integrate GEOIP2 for country code in response header with ingress

### DISCUSSION THREAD

**Discussion by fairok** [k3s-io/k3s#5572] [Q&A]
**Is your feature request related to a problem? Please describe.**

No.
**Describe the solution you'd like**

K3S integrated traefik ingress as default, is there any way to use GEOIP2 to add the country code to the response header?
**Describe alternatives you've considered**

Install the GEOIP2 as plugin.
**Additional context**

No.
**Backporting**

- [ ] Needs backporting to older releases

**Comment by brandond** [comment]
This sounds like a good question to take to the traefik repo. There is nothing unique about the version of traefik packaged in K3s: https://github.com/traefik/traefik/issues/4868
